app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/BrowserFragmentTest.kt

```diff
@@ -25,6 +31,23 @@ class BrowserFragmentTest {
     @Before
     fun setUp() {
         Intents.init()
+
+        LocalDatabaseProvider.setDatabase(
+            ApplicationProvider.getApplicationContext(),
+            LocalDatabaseProvider.CARDS_DATABASE_NAME,
+            true
+        )
+        LocalDatabaseProvider.debugging = true

```

Comment:
debugging is not the solution to simulate a mock data.
https://discord.com/channels/1076146742615097397/1102641526053548044

Reply:
Thanks I just did not have time to change it since the question was not answered yet